tools/docker/functions.sh

```diff
@@ -16,6 +16,10 @@ status() {
 	printf '\033[1;35m'"$@\n"'\033[0m'
 }
 
+mac_to_hex(){
+   echo "0x$(echo $1 | tr --delete :)"
```

Comment:
Sorry, I still have a slight improvement:
```suggestion
   echo "0x$1" | tr --delete :
```

It does mean the smiley is gone :-(